js/qm-numbers-rating.js

```diff
@@ -0,0 +1,83 @@
+jQuery(document).ready(function () {
+
+    console.debug({
+        variableName: qmShortCodeDefinedVariable,
+        isNegative: qmShortCodeDefinedNegative,
+        showLabels: qmShortCodeDefinedShowLabels
+    });
+
+    if (qmShortCodeDefinedShowLabels === 'false') {
+        jQuery('.rating-button-label').hide();
+    }
+
+    var ratingButtons = jQuery('.rating-button');
+
+    var colorClasses = ['qm-black', 'qm-blue', 'qm-green', 'qm-yellow', 'qm-red'];
+
+    for (var i = 0; i < ratingButtons.length; i++) {
+        var button = ratingButtons[i];
+
+        if (qmShortCodeDefinedNegative === 'true') {
+            jQuery(button).addClass(colorClasses[colorClasses.length - i - 1]);
+        } else {
+            jQuery(button).addClass(colorClasses[i]);
+        }
+
+    }
+
+    Quantimodo.getVariableByName(qmShortCodeDefinedVariable, function (variable) {
+        if (variable && variable.abbreviatedUnitName == '/5') {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1049789498/issues/source?bid=2836188&fileBranchId=2883147#l29)